k9mail/src/main/java/com/fsck/k9/mailstore/migrations/MigrationTo42.java

```diff
@@ -1,25 +1,26 @@
 package com.fsck.k9.mailstore.migrations;
 
 
-import java.util.List;
-
-import timber.log.Timber;
+import android.os.SystemClock;
 
-import com.fsck.k9.K9;
 import com.fsck.k9.mail.Folder;
 import com.fsck.k9.mailstore.LocalFolder;
 import com.fsck.k9.mailstore.LocalStore;
 import com.fsck.k9.preferences.Storage;
 import com.fsck.k9.preferences.StorageEditor;
 
+import java.util.List;
+
+import timber.log.Timber;
+
 
 class MigrationTo42 {
     public static void from41MoveFolderPreferences(MigrationsHelper migrationsHelper) {
         try {
             LocalStore localStore = migrationsHelper.getLocalStore();
             Storage storage = migrationsHelper.getStorage();
 
-            long startTime = System.currentTimeMillis();
+            long startTime = SystemClock.elapsedRealtime();
```

Comment:
You've changed this one but reverted the following one in the rebase. This will break